cmd/collector/app/sanitizer/hash_sanitizer.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2022 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package sanitizer
+
+import (
+	"github.com/jaegertracing/jaeger/model"
+)
+
+// NewHashingSanitizer creates a sanitizer to add hash field to spans
+func NewHashingSanitizer() SanitizeSpan {
```

Comment:
```suggestion
func AddHashTag() SanitizeSpan {
```
and similarly file name `add_hash_tag`